Revamp dbaas documentation (#991)
percona/pmm-doc: docs/dbaas/databases.md

    - *Connection*:
        - *Host*: The hostname.
        - *Port*: The port number.
        - *Username*: The connection username.
        - *Password*: The connection password (click the eye icon <i class="uil uil-eye"></i> to reveal).
    - *DB Cluster Parameters*:
        - *K8s cluster name*: The Kubernetes cluster name.
        - *CPU*: The number of CPUs allocated to the cluster.
        - *Memory*: The amount of memory allocated to the cluster.
        - *Disk*: The amount of disk space allocated to the cluster.
    - *Cluster Status*:
        - *PENDING*: The cluster is being created.
        - *ACTIVE*: The cluster is active.
        - *FAILED*: The cluster could not be created.
        - *DELETING*: The cluster is being deleted.
        - *UPDATING*: The cluster is being updated.

### Delete a DB Cluster

1. Find the row with the database cluster you want to delete.

2. In the *Actions* column, open the <i class="uil uil-ellipsis-v"></i> menu and click *Delete*.

3. Confirm the action by clicking *Proceed*, or abandon by clicking *Cancel*.

    ![!](../_images/PMM_DBaaS_DB_Cluster_Delete.png)

!!! danger alert alert-danger "Danger"

>     Deleting a cluster in this way also deletes any attached volumes.

Reviewer: ```suggestion
    Deleting a cluster also deletes any attached volumes.  Please proceed with caution. 
```
